webapp/src/shared/components/Avatar/Avatar.tsx

```diff
@@ -7,20 +7,40 @@ import { Color } from '../../types';
 export type AvatarProps = {
   url: string;
   status?: StatusVariant;
+  XCoordinate?: number;
+  YCoordinate?: number;
 };
 
 type LargeAvatarProps = AvatarProps & {
   edit?: boolean;
   caption?: string;
 };
 
-export function SmallAvatar({ url, status }: AvatarProps) {
+export function SmallAvatar({
+  url,
+  status,
+  XCoordinate,
+  YCoordinate,
+}: AvatarProps) {
+  const position =
+    XCoordinate && YCoordinate
+      ? {
+          objectPosition: `${XCoordinate * 0.139}px ${YCoordinate * 0.139}px`,
```

Comment:
These strange numbers were found experimentally via trial and error: I tried to see the same image (for usefulness I used a chess like grid, with different numbers and colors on each cell) in the three sizes of Avatar we have, 
so that they would look the same way with the same x y coordinates. These coefficients make the three be in sync. When you set the position from the editable Avatar component, you can freely pass the same coordinates to Avatar small and large, and they'll look just like editable Avatar.